src/components/itemBox/ItemBox.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import { ITooltipProps, Tooltip } from '../tooltip/Tooltip';
+import { Content, IContentProps } from '../content/Content';
+
+export interface IItemBoxProps {
+  value: string;
+  displayValue?: string;
+  selected?: boolean;
+  active?: boolean;
+  hidden?: boolean;
+  disabled?: boolean;
+  divider?: boolean;
+  tooltip?: ITooltipProps;
+  classes?: string[];
+  prepend?: IContentProps;
+  append?: IContentProps;
+  onOptionClick?: (option: IItemBoxProps) => void;
+}
+
+export class ItemBox extends React.Component<IItemBoxProps, void> {
+
+  static defaultProps: Partial<IItemBoxProps> = {
+    tooltip: {
+      title: '',
+    },
+  };
+
+  getClasses(): string {
+    return classNames('item-box',
+      {
+        'selected': this.props.selected,
+        'active': this.props.active,
+        'disabled': this.props.disabled,
+        'hidden': this.props.hidden,
+        'divider': this.props.divider,
+      },
+      this.props.classes);
+  }
+
+  handleOnOptionClick = () => {
+    if (this.props.onOptionClick) {
+      this.props.onOptionClick(this.props);
+    }
+  }
+
+  render() {
+    return (
+      <Tooltip {...this.props.tooltip}>
+        <li
+          className={this.getClasses()}
+          onMouseDown={() => this.handleOnOptionClick()}
```

Comment:
onClick instead of mouse down?

Reply:
I got a problem with the onClick when we try to use it with other component like dropdown. Its a problem with the event order with the onBlur